README.md

```diff
@@ -1 +1,172 @@
-# nba-games
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1 style="border-bottom: none;">NBA TEAMS STATS</h1>
+</div>
+
+<div align="center">
+  <img src="./readme.png" alt="Screenshot 1" width="300" style="display: inline-block; margin-right: 10px;">
+
+</div>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 NBA-TEAMS-STATS ](#-NBA-TEAMS-STATS-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 NBA-TEAMS-STATS <a name="about-project"></a>
+
+**NBA-TEAMS-STATS** Is a website that provides users with the standing of NBA teams with their latest statistics accumulated during the 2019-2020 season
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://tailwindcss.com/docs/guides/create-react-app">Tailwind Css</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **List of NBA Teams with current conerence position**
+- **Statistics of each NBA team** 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://dev--marvelous-mandazi-c7fe36.netlify.app/)
+- [video walkthrough Link](https://www.loom.com/share/30fdc28e49524da69ffb00842d47eb26?sid=f617c166-ce16-4cd9-8177-1215b94615a4)
```

Comment:
Nice Job so far, your video presentation looks great👍🏼. But you seem to have forgotten to make mention of a piece of code or feature in the application that you implemented and you are proud of😉. Kindly do well to talk about that as this is a requirement in your video presentation👍🏼.

<img width="1010" alt="image" src="https://github.com/migo1/nba-teams-stats/assets/64643995/ac04ac58-d640-425f-a521-01ee3c613a46">